common/governments/civics/hfe_civic_pirate.txt

```diff
@@ -127,12 +263,28 @@ hfe_civic_pirate_space_mining = {
 	potential = {
 		country_type = {
 			value = hfe_pirate
+			NOT = { value = default }
+		}
+	}
+	possible = {
+		country_type = {
+			value = hfe_pirate
+			NOT = { value = default }
 		}
 	}
 	random_weight = {
 		base = 1000
+#		modifier = {
+#			factor = 0
+#			NOT = { is_country_type = hfe_pirate }
+#		}
+#		modifier = {
+#			factor = 0
+#			is_country_type = default
+#		}
 	}
 	modifier = {
-		stations_produces_mult = 1
+		stations_produces_mult = 1.5
+		country_trust_growth = 0.05
```

Comment:
In the very distant future, I plan on trying to write a Galactic Resolution in which the Galactic Community decides whether or not Pirates should be allowed to join the Galactic Community. This is in anticipation of that change.

Reply:
galactic community still have not use of trust

Reply:
Consider it flavor. Pirates do have relations and opinions, remember. The Asteroid Miners don't attack people very often, so other people have a little more tolerance of them. Besides, if Paradox ever allows for dynamic diplomatic buttons, this will come in handy.